packages/store/src/types.js

```diff
@@ -1,96 +1,19 @@
 /// <reference types="ses"/>
 
-/** @typedef {import('@endo/marshal').Passable} Passable */
-/** @typedef {import('@endo/marshal').PassStyle} PassStyle */
-/** @typedef {import('@endo/marshal').CopyTagged} CopyTagged */
-/** @template T @typedef {import('@endo/marshal').CopyRecord<T>} CopyRecord */
-/** @template T @typedef {import('@endo/marshal').CopyArray<T>} CopyArray */
-/** @typedef {import('@endo/marshal').Checker} Checker */
-/** @typedef {import('@endo/marshal/src/rankOrder').RankCompare} RankCompare */
-/** @typedef {import('@endo/marshal/src/rankOrder').RankComparison} RankComparison */
-
-// /////////////////////////////////////////////////////////////////////////////
-// Placeholder redundant types, to be imported from `@endo/patterns` instead.
-
-// TODO placeholder. Figure out how to import from `@endo/patterns` instead
-/** @typedef {Passable} Key */
-
-// TODO placeholder. Figure out how to import from `@endo/patterns` instead
-/** @typedef {Passable} Pattern */
-
-// TODO placeholder. Figure out how to import from `@endo/patterns` instead
+/** @typedef {import('@endo/pass-style').Passable} Passable */
```

Comment:
note that this is currently `any`. IMO while there the documentation value of naming it, it misleads the user into thinking they're getting some type annotation when in fact it's erasing any type information by propagating `any`.

Out of scope for this PR though. https://github.com/endojs/endo/issues/1488